src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({
+  person,
+  partner = person.isMarried
+    ? `${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`
+    : 'I am not married',
```

Comment:
Redundant, you already have all needed information inside person object and then you can calculate person partner inside Person component based on person sex